.gitignore

```diff
@@ -16,6 +16,7 @@ hs_err_pid*
 /target/
 /output/
 /bin/
+/test/
```

Comment:
Why is this added? Usually test/input/... is something very useful to store the files for regression tests.